### DISCUSSION TITLE
Feeding Breakdown like Diaper Changes

### DISCUSSION THREAD

**Discussion by TaddyJ** [babybuddy/babybuddy#366] [Ideas]
Currently my wife and I are doing a mix of Formula and Breast milk for a few reasons.

She has asked me if there is a way I can get a breakdown similar to the diaper change graphic to show the differences between what we've been feeding our son between the two.

Would be fairly easy I would imagine to have a similar style graphic with the different feeding types and the volumes in it on a daily rotation?

**Comment by cdubz** [comment]
I think this make sense. Are you thinking like a stacked graph of volumes by type, or something else? Are we concerned about _number_ of type of feeding, or just volume of different types?

**Reply by TaddyJ** [reply]
Just the same as the way the diapers are visually stacked I think is a good start right now I think would be handy as per request of my wife! :)
Maybe in the sections of it could be say [Breast 4x 360mls | Formula 4x 360mls] each bit being coloured is nice for a quick glance.

I think the types of feeding is fine for us right now. I did notice another person mention tube feeding which would have been handy while our bub was in NICU as for several weeks all his feeds were through tubes and then we transitioned away.

Overall we are very happy with it! :)
